### PR TITLE
Remove unused GraphQL client/fetch from site preview handlers

### PR DESCRIPTION
**Pull request by fraxachun**

### CODE REVIEW

File: packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export type SitePreviewParams = {
     previewData?: SitePreviewData;
 };
 
-export async function sitePreviewRoute(request: NextRequest, _graphQLFetch: unknown /* deprecated: remove argument in v8 */) {
+export async function sitePreviewRoute(request: NextRequest) {
```
Comment (manuelblum) [inline packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts:20]: i would also prefer and recommand to have an codemod for removing of this argument.

**Review by johnnyomair — COMMENTED** [review]
Please fix conflicts.

File: packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts (hunk 1 above)
Comment (johnnyomair) [reply packages/site/cms-site/src/sitePreview/SitePreviewUtils.ts:20]: Here you go: https://github.com/vivid-planet/comet-upgrade/pull/33

**Review by johnnyomair — COMMENTED** [review]
Please fix conflicts.

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**